src/solvers/dg_simplices/types_and_traits.jl

```diff
@@ -0,0 +1,17 @@
+
+# `MultiDG` refers to both multiple DG types (polynomial/SBP, simplices/quads/hexes) as well as
+# the use of multi-dimensional operators in the solver.
+const MultiDG{NDIMS, ElemType, ApproxType, SurfaceIntegral, VolumeIntegral} =
+  DG{<:RefElemData{NDIMS, ElemType, ApproxType}, Mortar, SurfaceIntegral, VolumeIntegral} where {Mortar}
+
+const MultiDGWeakForm{NDIMS, ElemType, ApproxType} =
+  MultiDG{NDIMS, ElemType, ApproxType, <:SurfaceIntegralWeakForm, <:VolumeIntegralWeakForm}
+
+const PolyDGFluxDiff{NDIMS, ElemType} =
+  MultiDG{NDIMS, ElemType, Polynomial, <:SurfaceIntegralWeakForm, <:VolumeIntegralFluxDifferencing} where {NDIMS, ElemType}
+
+const SBPDGFluxDiff{Dim, Elem} =
+  MultiDG{NDIMS, ElemType, <:SBP, <:SurfaceIntegralWeakForm, <:VolumeIntegralFluxDifferencing} where {NDIMS, ElemType}
+
```

Comment:
We do need better names for all of this - and the folders in `src`, `examples` etc. It is confusing that quads are included in a folder called simplices.

Reply:
How about renaming the folder `dg_simplices` (and similarly in `examples`) to `multi_dg`?

Reply:
What about changing the order for consistency with the other solvers: `dg_multi`? In the docs/comments, we could explain that this refers to
- multiple element types (triangles, tetrahedra, quads/hex, ...)
- multiple possible basis types (modal, SBP)
- multidimensional operators (SBP operators on triangles etc. are often called this way)

Reply:
Good idea! Does changing the type alias from `MultiDG` to `DGMulti` sound OK too?

Reply:
Sure, that's fine with me

Reply:
I've commented on this in another thread already before I saw this one. As stated there, now that it's called `dg_multi`, the type should also be `DGMulti` even if it looks slightly funny (to me, at least)